templates/invoice.html

```diff
@@ -0,0 +1,356 @@
+        }
+
+        .header-title {
+            display: block;
+            font-style: normal;
+            font-weight: 600;
+            font-size: 19px;
+            letter-spacing: 0.02em;
+            padding-bottom: 5px;
+            color: #28234A;
+        }
+
+        .content-padded {
+            padding: 20px;
+        }
+
+        .content-tight-padded {
+            padding: 0 20px 10px 20px;
+        }
+
+        .normal-text {
+            color: #28234A;
+        }
+
+        .summary-row {
+            line-height: normal;
+        }
+
+        .saleor-logo {
+            height: 40px;
+            width: 117px;
+            padding-bottom: 20px;
+        }
+
+        .products-table {
+            width: 100%;
+            line-height: 1.8;
+        }
+
+        .summary-table {
+            width: 100%;
+            line-height: 1.8;
+            padding-top: 20px;
+        }
+
+        .row-category > td,
+        .row-product > td {
+            border-bottom: 2px solid #EFF5F8;
+        }
+
+        .cell-product {
+            width: 55%;
+        }
+
+        .cell-price {
+            width: 15%;
+        }
+
+        .cell-summary {
+            width: 70%;
+            text-align: right;
+            padding-right: 30px;
+        }
+
+        .cell-quantity {
+            width: 15%;
+        }
+
+        .cell-total-price {
+            width: 15%;
+        }
+
+        .content-separator {
+            display: inline-block;
+            width: 100%;
+            border-bottom: 2px solid #EFF5F8;
+        }
+    </style>
+</head>
+
+<body>
+    <div class="section-header section-left">
+        <div class="content-padded">
+            <img class="saleor-logo" src="{{ logo_path }}" />
+            <br />
+            <span class="normal-text">{{ order.billing_address.company_name }}</span>
+            <br />
+            <span class="normal-text">{{ order.billing_address.street_address_1 }}</span>
+            <br />
+            {% if order.billing_address.street_address_2 %}
+            <span class="normal-text">{{ order.billing_address.street_address_2 }}</span>
+            <br />
+            {% endif %}
+            <span class="normal-text">{{ order.billing_address.city }}</span>
+            <br />
+            <span class="normal-text">{{ order.billing_address.postal_code }}</span>
+            <br />
+            <span class="normal-text">{{ order.billing_address.phone }}</span>
+            <br />
+            <span class="normal-text">{{ order.get_customer_email }}</span>
+            <br />
+        </div>
+    </div>
+    <div class="section-invoice-info section-right">
+        <div class="content-padded">
+            <span class="header-title">
+                {% trans "INVOICE INFORMATION" context "Invoice PDF header" %}
+            </span>
+            <br />
+            <span class="header-category">
+                {% trans "Order:" context "Invoice PDF header" %}
```

Comment:
```suggestion
                {% trans "Order:" context "Invoice PDF header: order number" %}
```

Helpful for translating